py3status/modules/sysdata.py

```diff
@@ -326,6 +348,14 @@ def sysData(self):
             self.values['swap_unit'] = swap_unit
             self.py3.threshold_get_color(swap_used_percent, 'swap')
 
+        # get load average
+        if self.py3.format_contains(self.format, 'load*'):
+            load1, load5, load15 = self.data.load()
+            self.values['load1'] = load1
+            self.values['load5'] = load5
+            self.values['load15'] = load15
+            self.py3.threshold_get_color(load1, 'load')
```

Comment:
I think this should be `self.py3.threshold_get_color(load1 * 100, 'load')` that way we can give thresholds using whole numbers eg `thresholds = [(0, "#FF0000"), (30, "#FFFF00"), (80, "#00FF00"),]`

Other than this I think we're looking good